### DISCUSSION TITLE
can support 1.21?

### DISCUSSION THREAD

**Discussion by sky-3311** [Kas-tle/java2bedrock.sh#32212] [Feature Requests]
hope support 1.21

**Comment by terraquad** [comment]
I would like this to be implemented as well, as [Stellarity](https://modrinth.com/resourcepack/stellarity-rp) 4.0.0+ uses item model components to work, which causes this [Bedrock Port](https://www.planetminecraft.com/texture-pack/stellarity-geysermc-pack-6274565/) to not work anymore.
